reana_workflow_controller/rest.py

```diff
@@ -1688,6 +1688,89 @@ def _stop_workflow(workflow):
         raise REANAWorkflowControllerError(message)
 
 
+def _run_yadage_workflow_from_spec(workflow):
```

Comment:
Why do we add these functions? Don't we start workflows only with `_start_workflow` now?